WolframLanguage.sublime-syntax

```diff
@@ -278,8 +357,22 @@ contexts:
             - include: sequence
         - match: (?=[^\[])
           pop: true
-    - match: ((?:{{symbol}}`)*(\${{symbol}}))(?=\s*:?=)
```

Comment:
This rule was here to add to the index global variable declarations
```
$GlobalValue = xxx;
```

Reply:
But this rule was not grammatical. System global varibles are defined by "built-in" variables. General varibles begin with "$" are not necessarily a "constant".